sass/tji/_general.scss

```diff
@@ -228,6 +225,26 @@ blockquote {
 	a {
 		font-family: $headingFont;
 	}
+	
+	@media (min-width: 37.5em) {
+		#primary-menu {
+			>li:last-of-type {
+				a {
+			    background-color: $primaryRed;
+			    padding: 10px 30px;
+			    border-radius: 5px;
+			    margin-top: -10px;
+			    color: #fff;
+			    box-shadow: 1px 1px 3px rgba(64,64,64,0.5);
+			    transition: all 0.35s;
+			    
+			    &:hover {
+				    background-color: $primaryGreen;
```

Comment:
do we need to add another color? Can we just make this button fade to blue so our color palette stays consistent?

Reply:
Nit: Also can you fix the tabbing?